dashboard/src2/objects/bench.js

```diff
@@ -534,6 +536,133 @@ export default {
 					}
 				}
 			},
+			{
+				label: 'Environment Variable',
+				icon: icon('tool'),
+				route: 'bench-environment-variable',
+				type: 'list',
+				list: {
+					doctype: 'Release Group Variable',
+					filters: releaseGroup => {
+						return {
+							parenttype: 'Release Group',
+							parent: releaseGroup.name
+						};
+					},
+					orderBy: 'creation desc',
+					fields: ['name'],
+					pageLength: 999,
```

Comment:
```suggestion
```
I've set pageLength to 999 to avoid an issue where the preview was not complete. We don't need a preview here as mentioned above so let's remove the line.